src/SpeciesDistributionToolkit.jl

```diff
@@ -11,6 +11,7 @@ using Reexport
 # Expose the components
 @reexport using OccurrencesInterface
 @reexport using SimpleSDMDatasets
+# @reexport using SimpleSDMPolygons  TO BE UNCOMMENTED AFTER PKG REGISTRATION
```

Comment:
drop this line, we'll do it when it's registered

Reply:
done